tests/odm/test_encoder.py

```diff
@@ -125,3 +128,30 @@ def test_keep_nulls_false():
     encoder = Encoder(keep_nulls=False, to_db=True)
     encoded_doc = encoder.encode(doc)
     assert encoded_doc == {"m": {"i": 10}}
+
+
+async def test_url():
+    # same logic from encoder.py IS_PYDANTIC_V2
+    is_pydantic_v2 = int(pydantic.VERSION.split(".")[0]) >= 2
```

Comment:
It is better to use IS_PYDANTIC_V2 variable here

Reply:
Thanks, I'll use it.